index.html

```diff
@@ -236,19 +236,19 @@
 <h2>Navigation Timing</h2>
 
 <section id='performanceentry'>
-  <h3>Relation with the <a href="https://www.w3.org/TR/performance-timeline-2/#idl-def-PerformanceEntry">PerformanceEntry</a> interface</h3>
-  <p><a>PerformanceNavigationTiming</a> interface extends the following attributes of <a href="https://www.w3.org/TR/performance-timeline-2/#idl-def-PerformanceEntry">PerformanceEntry</a> interface:</p>
+  <h3>Relation with the <a href="https://www.w3.org/TR/performance-timeline-2/#dom-performanceentry">PerformanceEntry</a> interface</h3>
+  <p><a>PerformanceNavigationTiming</a> interface extends the following attributes of <a href="https://www.w3.org/TR/performance-timeline-2/#dom-performanceentry">PerformanceEntry</a> interface:</p>
   <ul>
     <li>The <dfn id='dom-PerformanceNavigationTiming-name'><code>name</code></dfn> attribute MUST return the <code>DOMString</code> "<code>document</code>".</li>
     <li>The <dfn id='dom-PerformanceNavigationTiming-entryType'><code>entryType</code></dfn> attribute MUST return the <code>DOMString</code> "<code id="perf-navigation">navigation</code>".</li>
     <li>The <dfn id='dom-PerformanceNavigationTiming-startTime'><code>startTime</code></dfn> attribute MUST return a <a href="https://www.w3.org/TR/hr-time/#dom-domhighrestimestamp">DOMHighResTimeStamp</a> with a time value of 0. [[!HR-TIME-2]]</li>
-    <li>The <dfn id='dom-PerformanceNavigationTiming-duration'><code>duration</code></dfn> attribute MUST return a <a href="https://www.w3.org/TR/hr-time/#dom-domhighrestimestamp">DOMHighResTimeStamp</a> equal to the difference between <a link-for="PerformanceNavigationTiming">loadEventEnd</a> and <a href="#dom-PerformanceNavigationTiming-starttime"><code>startTime</code></a>, respectively.</li>
+    <li>The <dfn id='dom-PerformanceNavigationTiming-duration'><code>duration</code></dfn> attribute MUST return a <a href="https://www.w3.org/TR/hr-time/#dom-domhighrestimestamp">DOMHighResTimeStamp</a> equal to the difference between <a link-for="PerformanceNavigationTiming">loadEventEnd</a> and <a href="#dom-PerformanceNavigationTiming-startTime"><code>startTime</code></a>, respectively.</li>
   </ul>
 </section>
 
 <section id='PerformanceResourceTiming'>
-  <h3>Relation with the <a href="https://www.w3.org/TR/resource-timing/#idl-def-PerformanceResourceTiming">PerformanceResourceTiming</a> interface</h3>
-  <p><a>PerformanceNavigationTiming</a> interface extends the following attributes of the <a href="https://www.w3.org/TR/resource-timing/#idl-def-PerformanceResourceTiming">PerformanceResourceTiming</a> interface:</p>
+  <h3>Relation with the <a href="https://www.w3.org/TR/resource-timing/#dom-performanceresourcetiming">PerformanceResourceTiming</a> interface</h3>

```

Comment:
/TR/resource-timing/ -> /TR/resource-timing-2/

There's a few more places in the doc that we need to make the same change.